client/src/pages/Register.tsx

```diff
@@ -12,7 +13,7 @@ export interface RegisterFormValues {
 function Register() {
   return (
     <>
-      <div className="card w-96 bg-base-100 shadow-xl">
+      <div className="card w-full md:w-96 rounded-none md:rounded bg-secondary md:bg-base-100 md:shadow-xl">
```

Comment:
If you get rid of one of the `h-full` classes on the `RootLayout` component then you won't need to add all of this, it'll fix the issue with the card getting cut off on mobile landscape view